src/encode_task_tss_enrich.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+# ENCODE DCC TSS enrich wrapper
+# Author: Daniel Kim, Jin Lee (leepc12@gmail.com)
+
+import sys
+import os
+import argparse
+from encode_lib_common import strip_ext_bam, ls_l, log, logging, mkdir_p, rm_f
+from encode_lib_genomic import remove_read_group, samtools_index
+import matplotlib as mpl
+mpl.use('Agg')
+
+from matplotlib import pyplot as plt
+from matplotlib import mlab
+import metaseq
+import numpy as np
+import pybedtools
+import warnings
+warnings.filterwarnings("ignore")
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(prog='ENCODE TSS enrichment.')
+    parser.add_argument('--read-len-log', type=str, help='Read length log file (from aligner task).')
+    parser.add_argument('--nodup-bam', type=str, help='Raw BAM file (from task filter).')
+    parser.add_argument('--chrsz', type=str, help='2-col chromosome sizes file.')
+    parser.add_argument('--tss', type=str, help='TSS definition bed file.')
+    parser.add_argument('--out-dir', default='', type=str, help='Output directory.')
+    parser.add_argument('--log-level', default='INFO', help='Log level',
+                        choices=['NOTSET','DEBUG','INFO','WARNING','CRITICAL','ERROR','CRITICAL'])
+    args = parser.parse_args()
+    log.setLevel(args.log_level)
+    log.info(sys.argv)
+    return args
+
+
+def make_tss_plot(bam_file, tss, prefix, chromsizes, read_len, bins=400, bp_edge=2000,
+                  processes=8, greenleaf_norm=True):
+    '''
+    Take bootstraps, generate tss plots, and get a mean and
```

Comment:
Mentioning the filenames that are written as side-effect of this function would be good.